pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -552,20 +552,20 @@ func (c *Cluster) startOSDDaemonsOnNode(nodeName string, config *provisionConfig
 		if createErr != nil {
 			if !kerrors.IsAlreadyExists(createErr) {
 				// we failed to create job, update the orchestration status for this node
-				logger.Warningf("failed to create osd deployment for node %s, osd %v: %+v", n.Name, osd, createErr)
+				logger.Warningf("failed to create osd deployment for node %q, osd %v: %v", n.Name, osd, createErr)
```

Comment:
It might be useful to print the struct fields (`%+v`) for the osd?
```suggestion
				logger.Warningf("failed to create osd deployment for node %q, osd %+v: %v", n.Name, osd, createErr)
```